src/core/components/model.jsx

```diff
@@ -23,7 +23,7 @@ export default class Model extends ImmutablePureComponent {
       return ref.replace(/^.*#\/definitions\//, "")
     }
     if ( ref.indexOf("#/components/schemas/") !== -1 ) {
-      return ref.replace("#/components/schemas/", "")
+      return ref.replace(/^.*#\/components\/schemas\//, "")
```

Comment:
**Change request**: merge the latest changes on `facelift` into this branch, so that this doesn't show up in the diff.

(This change was introduced in #5334 and pulled into `facelift` via #5370)

Reply:
Done